### ISSUE TITLE
next/image not loading image only used on media=print

### ISSUE THREAD

**Issue by HelderFSFerreira** [vercel/next.js#24691]
### What version of Next.js are you using?

latest

### What version of Node.js are you using?

v14.15.1

### What browser are you using?

Chrome and Safari (tested)

### What operating system are you using?

macOS

### How are you deploying your application?

Vercel

### Describe the Bug

I have a component with an image that should only be visible when the page is about to print (media = print) but when I try to print directly after I entered the page it never renders.

The only way I have to render the image by debug (on Chrome), changing the media type to print, this way I get the rendered image. Only after that I can print the page correctly.

### Expected Behavior

Have the image rendered when the user decides to print the page.

### To Reproduce

Can see the code at this  [repo](https://github.com/HelderFSFerreira/gravidasv2) and use [this](https://gravidasv2-git-optimizefontsize-helderfsferreira.vercel.app) vercel deployment to reproduce the issue .

I could only get it working using the priority property. You can review it [here](https://github.com/HelderFSFerreira/gravidasv2/pull/22)

To render the image I used this chrome setting:
![image](https://user-images.githubusercontent.com/18551619/116792350-fcaf0e00-aab7-11eb-9302-ec0ae82ad4b7.png)

**Comment by JanKaifer** [comment]
This issue is getting stale.
Could you check if it’s still happening in `next@canary`?

**Comment by lachlanjc** [comment]
Not exactly the same repro, but after updating my images to the new component on Next 13, if I don't scroll down the page for them to load in my browser first, the images are blank when printing. https://2019.lachlanjc.com/music has a ton of images on a long page as a test case, running the new component. (Behavior was the same on the old component.)

**Comment by JanKaifer** [comment]
Hmm, all the images load for me just fine on that website (except for some weird modal in the middle). but that is there even after scrolling.

Could you send minimal repro? Ideally with source code.

**Comment by balazsorban44** [comment]
This issue has been automatically closed because it wasn't verified against next@canary. If you think it was closed by accident, please leave a comment. If you are running into a similar issue, please open a new issue with a reproduction. Thank you.

**Comment by HelderFSFerreira** [comment]
It is fixed for me
